deployments/warp_routes/FORM/ethereum-form-config.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
+  - addressOrDenom: "0xEa35E107fB64Ff8C3c8c37DB2d23A7179f31D8EC"
+    chainName: ethereum
+    collateralAddressOrDenom: "0xE7deE4823EE18F1347F1Cf7997f70B94eFDe2E1F"
```

Comment:
is there a coingecko ID yet for FORM? I assume not? if there is one can you add it

Reply:
ya I checked, the token isn't live yet.